released/packages/coq-monae/coq-monae.0.3.1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/monae"
+dev-repo: "git+https://github.com/affeldt-aist/monae.git"
+bug-reports: "https://github.com/affeldt-aist/monae/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "Monads and equational reasoning in Coq"
+description: """
+This Coq library contains a hierarchy of monads with their laws used
+in several examples of monadic equational reasoning."""
+
+build: [
+  [make "-j%{jobs}%"]
+  [make "sect5"]
+  [make "-C" "impredicative_set"]
```

Comment:
I have been wondering. Are those supposed to be tests or examples that will not get installed? If so, you could tag them as such:

    [make "-C" "impredicative_set"] {with-test}

Reply:
It is supposed to be part of the distribution. There is a subdirectory that mimics a part of the top directory but provides slightly different results thanks to impredicative Set.

Reply:
Just to be clear. Are these files installed by the `make install` command below? If so, disregard my comment.

Reply:
It looks like they aren't, which is not intended. :-/

Reply:
I guess I need to change the Makefile to add some new custom install entry to do both install of the root and install of the subdirectory, and change the opam file to call this entry instead of the default install produced by coq_makefile, but this will be for the next release.